source/_integrations/enocean.markdown

```diff
@@ -405,7 +405,7 @@ sender_id:
 driving_time:
   description: The Time your shutter needs to drive from totally open to totally close in seconds.
   required: true
-  type: int
+  type: string
```

Comment:
This seems incorrect. Why should this be a string?

Reply:
Yes, you are right.
I have done the changes.
Thanks for reading